src/lib/address_resolve/tests/TestAddressResolve_DefaultImpl.cpp

```diff
@@ -18,24 +18,52 @@
 
 #include <lib/address_resolve/AddressResolve_DefaultImpl.h>
 #include <lib/core/StringBuilderAdapters.h>
+#include <lib/dnssd/IPAddressSorter.h>
+#include <lib/support/StringBuilder.h>
+#include <transport/raw/PeerAddress.h>
 
 using namespace chip;
 using namespace chip::AddressResolve;
 
+namespace pw {
+
+template <>
+StatusWithSize ToString<Transport::PeerAddress>(const Transport::PeerAddress & addr, pw::span<char> buffer)
+{
+    char buff[Transport::PeerAddress::kMaxToStringSize];
+    addr.ToString(buff);
+    return pw::string::Format(buffer, "IP<%s>", buff);
+}
+
+} // namespace pw
+
 namespace {
 
 using chip::Dnssd::IPAddressSorter::IpScore;
 using chip::Dnssd::IPAddressSorter::ScoreIpAddress;
 
 constexpr uint8_t kNumberOfAvailableSlots = CHIP_CONFIG_MDNS_RESOLVE_LOOKUP_RESULTS;
 
-Transport::PeerAddress GetAddressWithLowScore(uint16_t port = CHIP_PORT, Inet::InterfaceId interfaceId = Inet::InterfaceId::Null())
+Transport::PeerAddress GetAddressWithLowScore(uint16_t idx = 4, uint16_t port = CHIP_PORT,
```

Comment:
Not sure I care because this is a test, so if the CI is already done and there are no other comments, please ignore. BUT, if you end up needing to do another CI run for some reason, if you could pop in a comment about what idx is here, that would be helpful to understand the test.

Reply:
Had a fix after all and updated comment.